README.md

```diff
@@ -18,10 +75,10 @@ You can assume that you have access to the following functions:
 - `get_recording()`: Gets the latest recording. 
 - `get_events(recording)`: Returns a list of `InputEvent` objects for the given recording.
 
-## Contributing 
+### Instructions
 
 1. Fork this repository and clone it to your local machine. 
-2. Get puterbot up and running by following the instructions under [Setup and Run puterbot](#setup-and-run-puterbot) below.
+2. Get puterbot up and running by following the instructions under [Setup and Run puterbot](#setup-and-run-puterbot).
```

Comment:
I think this should be [Setup](#setup) now.

Reply:
Resolved in 7969e98